.github/workflows/IOnIntegration_Integration.yml

```diff
@@ -0,0 +1,38 @@
+name: I-On Integration Integration
```

Comment:
What do you think about changing the name of the workflow to I-On Integration Quality? I'm not 100% satisfied with it but I think it is strange to have * Integration Integration *. Do you have any other alternative or do you think that this is a non-issue?

Reply:
Didn't even noticed 😅 
Maybe we could change it to Stating.
So we would implement/run/test locally and then deploy to the staging env where we would test the project as a whole.
Then production env would be production itself 😄

Reply:
Good idea! I agree with "I-On Integration Staging"